waku/waku_keystore/keyfile.nim

```diff
@@ -327,6 +327,22 @@ proc compareMac(m1: openArray[byte], m2: openArray[byte]): bool =
   else:
     return false
 
+type CypherParams = object
+  iv: string
+
+type CryptoNew = object
+  cipher: string
+  cipherparams: CypherParams
+  ciphertext: string
+  kdf: string
+  kdfparams: JsonNode
+  mac: string
+
+type KeystoreEntry = object
+  crypto: CryptoNew
+  id: string
+  version: string
+
```

Comment:
Nitpick and not urgent but I'd either put it at the top of the file or in another `keyfile_types.nim` file.